ide/che-core-ide-jseditor/src/main/java/org/eclipse/che/ide/jseditor/client/debug/BreakpointManagerImpl.java

```diff
@@ -324,4 +335,24 @@ public void onLineChange(final VirtualFile file, final int firstLine, final int
             unmarkCurrentBreakpoint();
         }
     }
+
+    private void onEditorOpened(VirtualFile file, EditorPartPresenter editor) {
+        final List<Breakpoint> fileBreakpoints = this.breakpoints.get(file.getPath());
+
+        if (fileBreakpoints != null) {
```

Comment:
Maybe it would be better to use return for reduction of nesting. I mean 
if (fileBreakpoints != null) {return;}